en/docs/develop/components/rest-api.md

```diff
@@ -36,56 +36,44 @@ Choreo allows you to design your REST API in three possible ways as shown in the
+WSO2 provides a [Ballerina extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=WSO2.ballerina) which you can use to eaasily develop your REST API as a Ballerina project.
 
-### Pro-code mode
-
-The pro-code mode of Choreo allows developers to directly edit code in a manner where the graphical representation in the low-code editor gets updated accordingly. Choreo also allows viewing the low-code and pro-code editors side-by-side.
+## Lifecycle of a REST API
 
-![Low code and pro-code view](../../assets/img/rest-apis/low-code-and-pro-code-view.png){.cInlineImage-full}
+Once you implement a REST API, it needs to be built, tested, and eventually put into production. You can apply any of the six API lifecycles states (i.e., created, pre-released, published, blocked, deprecated, and retired) while adhering to their respective lifecycle flows.
 
-Choreo gives developers the flexibility to use the pro-code mode only or to use it for implementing specific parts of the API while using the low-code mode for the rest, based on their preference. Choreo treats the source code (generated and handwritten) as the single source of truth and has no limitations or restrictions on the mode you can choose to implement API functionality.
+When you publish an API, you can also publish a connector for that API to make it available on the Choreo Marketplace.
 
-## Lifecycle of a REST API
-
-Once you implement a REST API, it needs to be built, tested, and eventually put into production. Choreo executes a CI/CD pipeline to manage the API lifecycle. Choreo stores the source code of APIs in a private GitHub repository for the user account. It also provides the capability for developers to connect their own GitHub repositories containing the source code of their APIs. The CI/CD pipeline in Choreo takes an API through the complete process, starting from code checkout to compile, build, test, and finally deploy.
 
 ## Deploying a REST API
 
 The following diagram illustrates the procedure to deploy an API in Choreo to the default development environment.
 
-![Deploy API](../../assets/img/rest-apis/deploy-api.png){.cInlineImage-full}
+![Deploy API](../../assets/img/rest-apis/deploy-api.png){.cInlineImage-small}
 
 Choreo runs a professional, enterprise-grade CI/CD process to deploy APIs to their runtime(data plane) clusters. The data plane of Choreo runs on a Kubernetes stack under the hood. Therefore, you can benefit from its features such as auto-scaling, auto-healing, secret-management, liveness and readiness checks, etc.
 
+Choreo provides the capability for developers to connect their own GitHub repositories containing the source code of their APIs. The CI/CD pipeline in Choreo takes an API through the complete process, starting from code checkout to compile, build, test, and finally deploy.
```

Comment:
Suggestion: "Choreo allows developers to connect their own GitHub repositories containing the source code of their APIs. The CI/CD pipeline in Choreo takes an API through the complete process, starting from code checkout to compiling, building, testing, and finally deploying."